DOCSP-8676 fix grammar and consistency (#51)
mongodb/docs-node: source/fundamentals/monitoring.txt

Event Descriptions
------------------

> The following table lists all the SDAM events that are available to your

Reviewer: This sentence sounds wrong to me. I would suggest:
```suggestion
The following table lists all the SDAM events made available to your
```

Author: Updated to be more direct:

> You can subscribe to any of the following SDAM events:

     - Description

   * - ``serverOpening``
     - Created when a connection to an instance opens.

   * - ``serverClosed``
     - Created when a connection to an instance closes.

   * - ``serverDescriptionChanged``
     - Created when an instance state changes (such as from secondary to
       primary).

   * - ``topologyOpening``

>      - Created prior to attempting a connection to an instance.

Reviewer: ```suggestion
     - Created prior to establishing a connection to an instance.
```

Author: I think the nuance communicated with "attempting" is that it could fail whereas establishing implies that it was successful. I could be more specific and use "attempting to establish a connection" if you think "attempting a connection" sounds awkward. Let me know what you think. @nathan-contino-mongo

Reviewer: I do think that "attempting to establish a connection" would sound better, but that's also a good chunk of words, so... "attempting" seems fine to me. Good point!
